html/index.html

```diff
@@ -14,6 +15,10 @@
             box-shadow: none;
             margin: 0;
         }
+
```

Comment:
Same comments as in the experiment page PR, I don't think this CSS will be applied unless you use styled mode.

Reply:
The css is applied here, which enables the download button to be triggered by clicking *on* text and icon. Apart from this comment, I have added a source link for the highcharts module and set the font for both button text and item menu as `15px`. As heatmap is using extra download options, I keep the item lists in the download menu here.